python/baseline/remote.py

```diff
@@ -40,17 +42,24 @@ def predict(self, examples):
         :param examples: The input examples
         :return: The outcomes
         """
-        import requests
+        import httplib, urllib
+        import json
```

Comment:
The `requests` import was inside to make the dependency optional. Given that `json` is in the standard library we can move this import to the top of the file